pkg/domain/infra/abi/containers_runlabel.go

```diff
@@ -36,6 +36,11 @@ func (ic *ContainerEngine) ContainerRunlabel(ctx context.Context, label string,
 		return err
 	}
 
+	if options.Display {
+		fmt.Printf("command: %s\n", strings.Join(append([]string{os.Args[0]}, cmd[1:]...), " "))
```

Comment:
To double check: is the `command: ` prefix what we had in pre 2.0 time?

Reply:
This code was cut and paste from the 1.9.0 base.